### ISSUE TITLE
LightsOut MLP action head has only one output

### ISSUE THREAD

**Issue by ruoyiqiao** [RajGhugare19/on-computation-and-rl#2]
Hi, thanks for sharing the code!

I noticed that action_size is appended to layer_sizes, but output_dim_1 is not passed:

https://github.com/RajGhugare19/on-computation-and-rl/blob/41acbe6759e4fbdeaabb037e241b96715df63e61/lightsout/ppo.py#L224

output_dim_1 defaults to 1 and sets the logits dimension:

https://github.com/RajGhugare19/on-computation-and-rl/blob/41acbe6759e4fbdeaabb037e241b96715df63e61/lightsout/models/ppo.py#L181-L201

I checked both environments: the logits have shape (batch_size, 1), so the policy always selects action 0.

In my reproduction, I changed only the MLP construction:
```diff
- MLPActorValue(layer_sizes=args.policy_hidden_sizes + [action_size])
+ MLPActorValue(
+     layer_sizes=args.policy_hidden_sizes,
+     output_dim_1=action_size,
+ )
```

With this change, my three-seed runs reached 94.53% / 99.61% training success on 3×3 / 4×5, compared with 4.69% / 0.39% using the original initialization.

Could you clarify whether the MLP results in Table 1 and Figure 4 used this code path, or a different version? Thanks!
